src/components/UserProfile/TeamsAndProjects/UserProjectsTable.jsx

```diff
@@ -332,7 +338,13 @@ const UserProjectsTable = React.memo(props => {
                       <td>{`${project.projectName}`}</td>
                       {props.edit && props.role && (
                         <td style={{ width: '103px' }}>
-                          <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+                          <div
+                            style={{
+                              display: 'flex',
+                              alignItems: 'center',
+                              justifyContent: 'center',
```

Comment:
As suggested by other members, how about creating a separate class to facilitate reusability of this style?